src/EventListener/StoreRefererListener.php

```diff
@@ -67,7 +67,7 @@ public function onKernelResponse(FilterResponseEvent $event)
      */
     private function storeBackendReferer(Request $request)
     {
-        if (!$this->canModifyBackendSession($request)) {
+        if ('contao_backend' !== $request->attributes->get('_route') || !$this->canModifyBackendSession($request)) {
```

Comment:
Because the method is called `canModifyBackendSession` and it previously was not the same `if` too. But that would obviously work either way…